### PR TITLE
Added test create New View with select radio button "My View"

### PR DESCRIPTION
**Pull request by olpolezhaeva**

### CODE REVIEW

**Comment by olpolezhaeva** [comment]
[AT_026.002 | My Views > Create new view with select label "My view"](https://trello.com/c/Mz9pwc9B/199-at026002-my-views-create-new-view-with-select-label-my-view)

File: src/test/java/CreateNewViewWithSelectLabelMyView.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class CreateNewViewWithSelectLabelMyView extends BaseTest {
+
+    public boolean isNameExist() {
+        getDriver().findElement(By.xpath("//a[@title='New View']")).click();
+        String nameOfView = "My new view";
```
Comment (GitForProjects) [inline src/test/java/CreateNewViewWithSelectLabelMyView.java:10]: You have similar variable created in your test. I think you can just pass that variable in the isNameExist method as an input parameter. 
Actually I do not understand why do you use separate method with hardcoded xpaths inside. You can path all of then as an input parameters and re-use this method in the future

Comment (olpolezhaeva) [reply src/test/java/CreateNewViewWithSelectLabelMyView.java]: Thanks for review and advices!

**Review by karinka83 — APPROVED** [review]

**Review by GitForProjects — APPROVED** [review]
looks good

**Pull request merged**